### DISCUSSION TITLE
Mullvad on 4.2

### DISCUSSION THREAD

**Discussion by grrrrr** [PrivSec-dev/privsec.dev#214] [General]
Some small tweaks could be made to the mullvad guide.

1.  Mullvad now has both debian and fedora repos so if you set that up instead of downloading rpm it easier to keep up dates
2. On 4.2 the steps feel not to work the same as on 4.1    Mullvad gui does not auto start when you start the proxyvm.  I needed to go to mullvad gui and turn on auto start and auto connect.

**Comment by TommyTran732** [comment]
1. Done in https://github.com/PrivSec-dev/privsec.dev/commit/5cb4ad0edeeb96f47a0783478440f39c9d1e4a66
2. After you enable launch on start up and auto-connect, the VPN should auto connect everytime the proxyvm boots.
